.travis.yml

```diff
@@ -17,16 +17,18 @@ matrix:
 before_script:
   - export NODE_OPTIONS=–max_old_space_size=8192
 script:
-# build Serge deployment
-- yarn build
-# run the linting drop out on failure
-- yarn lint || travis_terminate 1;
-# start serge
-- yarn start-server &
-# run the tests drop out on failure
-- yarn test || travis_terminate 1;
-# kill open jobs (especially the Serge server running in the background)
-- kill $(jobs -p) || true
+  # build Serge deployment
+  - yarn build
+  # Build the storybook package
```

Comment:
@foxleigh81 - we don't need to build storybook on Travis, do we?

Reply:
We don't NEED to but we should. If the storybook build fails, we want to know about it.